### DISCUSSION TITLE
Exceptions

### DISCUSSION THREAD

**Discussion by nothingness6** [zanllp/infinite-image-browsing#544] [Ideas]
Hi, thanks for the great extension. I hope there would be an exception function. Once I added Outputs folder which includes mov2mov folder. And it shows tons of images I generated for video process. Thus, I'd like to except the folder. 
Please add this function in the next update. 
Cheers!

**Comment by zanllp** [comment]
Do you want to exclude specific folders in the search function and walk mode?
